apps/android/ImageClassification/README.md

```diff
@@ -2,6 +2,10 @@
 
 1. Java, android-sdk and sdkmanager is already set at user's end
 2. User should have Linux QNN SDK in local machine.
+3. If models need to be downloaded during the execution of build_apk.py, qai-hub must be installed first according to the[ README.md in ai-hub-models](https://github.com/quic/ai-hub-models/blob/main/README.md).

```

Comment:
Could you also move the space before `README.md` outside of the link text. Thanks!

Reply:
This was my negligence, thank you for pointing out the mistake.